docs/sphinx/using/examples/multi_gpu_workflows.rst

```diff
@@ -53,74 +39,74 @@ NVIDIA GPUs vs CPUs:
+requires roughly 8 GB. Adding a few more qubits will quickly exceed the memory of as single GPU.  The `mqpu` backend 
+solved this problem by pooling the memory of multiple GPUs across multiple nodes to perform a single state vector simulation. 
+
+
+.. image:: images/mgpu.png
+
+
+If you have multiple GPUs, you can use the following command to run the simulation across :math:`n` GPUs. 
+
+
+:code:`mpiexec -np n python3 program.py --target nvidia --target-option mgpu`
+
+This code will execute in an MPI context and provide additional memory to simulation much larger state vectors.
+
 
-where 4 is the number of GPUs one has access to and ``test`` is the file
-name chosen.
+Batching Hamiltonian Terms (`mqpu`)
+------------------------------------
 
-The ``nvidia-mqpu`` target uses a statevector simulator to simulate execution 
-on each virtual QPU.
-The ``remote-mqpu`` platform allows to freely configure what backend is used 
-for each platform QPU. 
-For more information about the different platform targets, please take a look at
-:doc:`../backends/platform`.
+Multiple GPUs can also come in handy for cases where applications might benefit from multiple QPUs running asynchronously.  The `mqpu` backend uses multiple GPUs to simulate each QPU so you can test and accelerate quantum applications with parallelization.
+
+
+.. image:: images/mqpu.png
+
+The most simple example is Hamiltonian Batching. In this case, an expectation value of a large Hamiltonian is distributed across multiple simulated QPUs, where each QPUs evaluates a subset of the Hamiltonian terms. 
 
-Batching Hamiltonian Terms
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Expectation value computations of multi-term Hamiltonians can be
-asynchronously processed via the ``mqpu`` platform.
 
 .. image:: ../../applications/python/images/hsplit.png
 
-For workflows involving multiple GPUs, save the code below in a
-``filename.py`` file and execute via:
-``mpirun -np n python3 filename.py`` where ``n`` is an integer
-specifying the number of GPUs you have access to.
 
+The code below evaluates the expectation value of a random 100000 term Hamiltonian. A standard :code:`observe` call will run the program on a single GPU.  Adding the argument :code:`execution=cudaq.parallel.thread` or :code:`execution=cudaq.parallel.mpi` will automatically distribute the Hamiltonian terms across multiple GPUs on a single node or multiple GPUs on multiple nodes, respectively.  
+
+The code is executed with :code:`mpiexec -np n python3 program.py --target nvidia --target-option mqpu` where :math:`n` is the number of GPUs available. 
+
+   
 .. literalinclude:: ../../snippets/python/using/examples/multi_gpu_workflows/hamiltonian_batching.py
     :language: python
     :start-after: [Begin Docs]
     :end-before: [End Docs]
 
-.. parsed-literal::
 
-    mpi is initialized?  True
-    rank 0 num_ranks 1
 
+Circuit Batching (`mqpu`)
```

Comment:
I think this is a second example of mqpu and should be under the same section title of mqpu